examples/statistics/time_series_histogram.py

```diff
@@ -71,7 +71,7 @@
 for i in range(num_series):
     y_fine[i, :] = np.interp(x_fine, x, Y[i, :])
 y_fine = y_fine.flatten()
-x_fine = np.matlib.repmat(x_fine, num_series, 1).flatten()
+x_fine = matlib.repmat(x_fine, num_series, 1).flatten()
```

Comment:
```suggestion
x_fine = np.tile(x_fine, (num_series, 1)).flatten()
```
Remove repmat altogether?